res/ui/actions_form.py

```diff
@@ -237,131 +235,131 @@ def setupUi(self, ActionsForm):
+        self.queueLabelBottom.setTextInteractionFlags(QtCore.Qt.TextInteractionFlag.NoTextInteraction)
         self.queueLabelBottom.setObjectName("queueLabelBottom")
         self.horizontalLayout_17.addWidget(self.queueLabelBottom)
-        self.queueLabelBottomPos = QtWidgets.QLabel(self.queueGroup)
-        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Maximum, QtWidgets.QSizePolicy.Preferred)
+        self.queueLabelBottomPos = QtWidgets.QLabel(parent=self.queueGroup)
+        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Maximum, QtWidgets.QSizePolicy.Policy.Preferred)
         sizePolicy.setHorizontalStretch(0)
         sizePolicy.setVerticalStretch(0)
         sizePolicy.setHeightForWidth(self.queueLabelBottomPos.sizePolicy().hasHeightForWidth())
         self.queueLabelBottomPos.setSizePolicy(sizePolicy)
-        self.queueLabelBottomPos.setTextInteractionFlags(QtCore.Qt.NoTextInteraction)
+        self.queueLabelBottomPos.setTextInteractionFlags(QtCore.Qt.TextInteractionFlag.NoTextInteraction)
         self.queueLabelBottomPos.setObjectName("queueLabelBottomPos")
         self.horizontalLayout_17.addWidget(self.queueLabelBottomPos)
-        spacerItem5 = QtWidgets.QSpacerItem(0, 0, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
+        spacerItem5 = QtWidgets.QSpacerItem(0, 0, QtWidgets.QSizePolicy.Policy.Expanding, QtWidgets.QSizePolicy.Policy.Minimum)
         self.horizontalLayout_17.addItem(spacerItem5)
         self.gridLayout_9.addLayout(self.horizontalLayout_17, 0, 0, 1, 1)
-        self.frame_9 = QtWidgets.QFrame(self.queueGroup)
-        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Maximum)
+        self.frame_9 = QtWidgets.QFrame(parent=self.queueGroup)
+        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Maximum)
         sizePolicy.setHorizontalStretch(0)
         sizePolicy.setVerticalStretch(0)
         sizePolicy.setHeightForWidth(self.frame_9.sizePolicy().hasHeightForWidth())
         self.frame_9.setSizePolicy(sizePolicy)
         self.frame_9.setMinimumSize(QtCore.QSize(0, 38))
-        self.frame_9.setFrameShape(QtWidgets.QFrame.StyledPanel)
-        self.frame_9.setFrameShadow(QtWidgets.QFrame.Raised)
+        self.frame_9.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
+        self.frame_9.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
         self.frame_9.setObjectName("frame_9")
         self.horizontalLayout_20 = QtWidgets.QHBoxLayout(self.frame_9)
         self.horizontalLayout_20.setObjectName("horizontalLayout_20")
-        self.label_4 = QtWidgets.QLabel(self.frame_9)
+        self.label_4 = QtWidgets.QLabel(parent=self.frame_9)
         self.label_4.setObjectName("label_4")
         self.horizontalLayout_20.addWidget(self.label_4)
         self.queueRatioSlider = TipSlider(self.frame_9)
```

Comment:
note that here I manually changed the code that pyuic6 outputted. The TipSlider does not have the parent attribute.